src/components/CookiesBar.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {useTranslation} from 'react-i18next';
+import {useCookies} from 'react-cookie';
+
+import {Button} from '@material-ui/core';
+
+const CookiesBar = () => {
+  const [cookies, setCookie] = useCookies(['cookies_info']);
+  const {t} = useTranslation();
+
+  const acceptCookies = () => {
+    setCookie('cookies_info', 'true', {path: '/'});
+  };
+  return (
+    <>
+      {!cookies.cookies_info && (
+        <div class="fixed-bottom pt-3 pb-4 bg-white shadow-lg">
```

Comment:
z bootstrapa już mało gdzie korzystamy, przypuszczam że w niedalekiej przyszłości będziemy chcieli go wywalić, więc będzie to do zmiany